### PR TITLE
PROGRAMMES-5952 Adding vote priority

### PR DESCRIPTION
**Pull request by votemike**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-5952

Can be viewed at programmes/b006m8dq?__flush_cache=1

### CODE REVIEW

File: app/Resources/views/find_by_pid/tlec.html.twig [hunk 1]
```diff
@@ -13,7 +13,15 @@
 
     {# TODO BBC Live Promo #}
 
-    {# TODO Priority Vote block #}
+    {% if programme.getOption('brand_layout') == 'vote' and programme.getOption('ivote_block') %}
+        <div class="p-g-p">
+            <div class="grid leading-gap">
+                <div class="grid__item">
+                    {{ dump(programme.getOption('brand_layout'), programme.getOption('ivote_block')) }}
```
Comment (Tasiobg) [inline app/Resources/views/find_by_pid/tlec.html.twig]: Is this `dump()` right? isn't this only for debugging?

Comment (votemike) [reply app/Resources/views/find_by_pid/tlec.html.twig]: From what I can tell, the ticket it only to get the placeholder ready. Do you also think that's what the ticket is asking for?
I didn't know what else to put inside the div if we're not yet implementing the vote thing.

File: src/DsAmen/Organism/Map/MapPresenter.php [hunk 2]
```diff
@@ -288,9 +288,13 @@ private function showMiniMap(): bool
     {
         if ($this->request->query->has('__2016minimap')) {
             return (bool) $this->request->query->get('__2016minimap');
-        } else {
-            // if ($this->is2016BrandPage() && $this->isVotePriority() ) return true;
-            return filter_var($this->programme->getOption('brand_2016_layout_use_minimap'), FILTER_VALIDATE_BOOLEAN);
         }
+
+        if ($this->programme->getOption('brand_layout') === 'vote') {
```
Comment (BjornTwachtmann) [inline src/DsAmen/Organism/Map/MapPresenter.php]: If the TLEC template is checking for the presence of the programme-options ivote bit, that should probably be the same here too. People will probably delete the vote and forget to change the layout back.

File: app/Resources/views/find_by_pid/tlec.html.twig (hunk 1 above)
Comment (BjornTwachtmann) [reply app/Resources/views/find_by_pid/tlec.html.twig]: it's fine for now I think

Comment (Tasiobg) [reply app/Resources/views/find_by_pid/tlec.html.twig]: Sorry I didn't realised it was just a placeholder, in that case looks good

**Comment by BjornTwachtmann** [comment]
There's some extra markup and stuff for iVote in V2, but as we're not implementing iVote I don't think it's worth replicating. This is good as a place to put the testing code for the new module I think

File: src/DsAmen/Organism/Map/MapPresenter.php (hunk 2 above)
Comment (votemike) [reply src/DsAmen/Organism/Map/MapPresenter.php]: So check for both?

Comment (BjornTwachtmann) [reply src/DsAmen/Organism/Map/MapPresenter.php]: yep

Comment (BjornTwachtmann) [reply src/DsAmen/Organism/Map/MapPresenter.php]: I could probably have expressed that better :)

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/DsAmen/Organism/Map/MapPresenter.php [hunk 3]
```diff
@@ -288,9 +293,13 @@ private function showMiniMap(): bool
...
-        } else {
-            // if ($this->is2016BrandPage() && $this->isVotePriority() ) return true;
-            return filter_var($this->programme->getOption('brand_2016_layout_use_minimap'), FILTER_VALIDATE_BOOLEAN);
         }
+
+        if ($this->isVotePriority) {
+            return true;
+        }
+
+        // if ($this->is2016BrandPage()) return true;
```
Comment (votemike) [inline src/DsAmen/Organism/Map/MapPresenter.php]: Do we need to worry about this any more? Or is everything now the 2016 brand page?

**Comment by votemike** [comment]
Test me please

**Pull request merged**